packages/terra-cell-grid/src/terra-dev-site/doc/example/CellGridExample.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import CellGrid, {
+  Cell,
+} from 'terra-cell-grid'; // eslint-disable-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
```

Comment:
We should be able to remove this `eslint-disable-line`. These errors should be ignored with our eslint override rule: https://github.com/cerner/terra-core/blob/master/package.json#L29